client/src/component/Addvideo.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+function AddVideo({ videoData }) {
```

Comment:
In React, we always want to use arrow functions, not function declaration. I would use `const addVideo = () => {}` instead 😊